### PR TITLE
Add @IIYAMA12

### PR DESCRIPTION
**Pull request by IIYAMA12**

### CODE REVIEW

File: site/class-2-style/IIYAMA12/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    return item.data.age;
+}
+
+function getItemPopulation(item) { // get the population of an item
+    return item.population;
+}
+
+console.log("d3.csv");
+d3.csv('index.csv', getItemtype, render); // (url [[, row], callback]) https://github.com/d3/d3-request/blob/master/README.md#csv
+// row is a function which can be used to filter or map objects.
```
Comment (wooorm) [inline site/class-2-style/IIYAMA12/index.js]: This looks good, but could you remove your commented backup below?

Comment (IIYAMA12) [reply site/class-2-style/IIYAMA12/index.js]: Of course! :D

File: site/class-2-style/IIYAMA12/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (wooorm) [inline site/class-2-style/IIYAMA12/index.html]: Oh, and remove the double doctype!

Comment (wooorm) [reply site/class-2-style/IIYAMA12/index.html]: Tip: click on the pencil in the top-right corner of this file.

**Pull request merged**